### PR TITLE
LHC phase calibration 

### PR DESCRIPTION
**Pull request by AllaMaevskaya**

### CODE REVIEW

File: Detectors/FIT/FT0/calibration/testWorkflow/FT0TFProcessor-Workflow.cxx [hunk 1]
```diff
@@ -17,7 +17,7 @@
 #include "DataFormatsFT0/ChannelData.h"
 #include "DataFormatsFT0/Digit.h"
 #include "FT0Calibration/FT0CalibrationInfoObject.h"
-
+#include "Framework/Logger.h"
```
Comment (mslupeck) [inline Detectors/FIT/FT0/calibration/testWorkflow/FT0TFProcessor-Workflow.cxx]: Do you really need a Logger include in this file? I don't see any `LOG(..) << ...` commands here.

File: Detectors/FIT/FT0/calibration/testWorkflow/GlobalOffsetsCalibrationSpec.h [hunk 2]
```diff
@@ -9,38 +9,44 @@
 // granted to it by virtue of its status as an Intergovernmental Organization
```
Comment (mslupeck) [inline Detectors/FIT/FT0/calibration/testWorkflow/GlobalOffsetsCalibrationSpec.h:9]: In general, `testWorkflow` directory contains mostly production code (and only w few test-examples), so perhaps it should be renamed to just `workflow`?

**Review by mslupeck — DISMISSED** [review]
Hi Alla, 
To me, this looks good at a first glance (except for 2 minor comments). I've noticed you've also added or modified some FV0 time offset calibration lines in the `FITCalibration` directory. Was there something missing or broken, or it is just github diff, which somehow doesn't show well the actually modified lines?

File: Detectors/FIT/FT0/calibration/testWorkflow/FT0TFProcessor-Workflow.cxx (hunk 1 above)
Comment (AllaMaevskaya) [reply Detectors/FIT/FT0/calibration/testWorkflow/FT0TFProcessor-Workflow.cxx]: I added timeStart and timeEnd to all FT0 and FV0 objects

**Comment by AllaMaevskaya** [comment]
Error is not related to my code. Can it be merge?

**Pull request merged**